deis-cluster-coreos/azuredeploy.json

```diff
@@ -359,8 +366,19 @@
             "publisher": "[variables('imagePublisher')]",
             "offer": "[variables('imageOffer')]",
             "sku": "[variables('imageSKU')]",
-            "version": "latest"
+            "version": "647.2.0"
```

Comment:
More recent versions of CoreOS include a version of Docker which is known to have issues. We plan to update to the latest stable release as soon as a version with Docker 1.7.1+ makes it to the stable channel.